routing/unified_edges.go

```diff
@@ -277,16 +309,20 @@ func (u *edgeUnifier) getEdgeNetwork(amt lnwire.MilliSatoshi) *unifiedEdge {
 
 		// Use the policy that results in the highest fee for this
 		// specific amount.
-		fee := edge.policy.ComputeFee(amt)
+		fee := int64(edge.policy.ComputeFee(amtWithInboundFee))
```

Comment:
Not sure, but it looks like we would need to compute the fee based on the sum of inbound and outbound, otherwise we could have failing situations, for example a channel with (in base fee: 0msat, out base fee: 1msat) could get picked over a channel (in base fee: 2msat, out base fee 0msat), which would result in an insufficient fee error, right?

Reply:
You are right. Corrected this and also modified the test to cover the addition of the inbound fee.